charts/consul/templates/_helpers.tpl

```diff
@@ -371,13 +371,50 @@ Consul server environment variables for consul-k8s commands.
 {{- end -}}
 
 {{/*
-Fails global.cloud.enabled is true and global.cloud.secretName is nil or tempty.
+Fails global.cloud.enabled is true and one of the following secrets is nil or empty.
+- global.cloud.resourceId.secretName
+- global.cloud.clientId.secretName
+- global.cloud.clientSecret.secretName
 
-Usage: {{ template "consul.validateCloudConfiguration" . }}
+Usage: {{ template "consul.validaterequiredCloudSecretsExist" . }}
```

Comment:
changed name.  this used to be one secret with 6 six keys (3 required and 3 optional).  Now there are 6 secrets for more flexibility and the ability to allow optional values.